twilio/http/bearer_token_http_client.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+import jwt
+
+from twilio.base.version import Version
+from twilio.http.token_manager import TokenManager
+from twilio.twilio_bearer_token_auth import TwilioBearerTokenAuth
```

Comment:
where is this file twilio.twilio_bearer_token_auth? Can't seem to find it

Reply:
deleted this one